README.md

```diff
@@ -1,32 +1,52 @@
+Therefore, the goal of the HMG is to develop an advanced human model that integrates both skeletal and muscular information. This model does not only include detailed anatomical and inertial data but also it is scalable to accommodate the specific features of each human subject.
+
+<br>
 
 <p align="center">
-<img src= https://github.com/ami-iit/human-model-generator/assets/116801366/ffb6bbd9-632a-4201-b7ee-dcb99a23fda2 width ="600" height="500">
+<img src= https://github.com/ami-iit/human-model-generator/assets/118193358/095c6a28-5dd8-4a73-a1ab-d2d5446c2e39 width ="650" height="500">
 </p>
 
-## Dependencies
-- [`python3`](https://wiki.python.org/moin/BeginnersGuide)
-- [`urdf-modifiers`](https://github.com/icub-tech-iit/urdf-modifiers)
+</br>
+
+## Dependencies 
+This library requires the following dependencies:
+
+- [``idyntree``](https://github.com/robotology/idyntree)
+- [``urdf-modifiers``](https://github.com/icub-tech-iit/urdf-modifiers)
+
+## Installation with [conda](https://docs.conda.io/en/latest/) (recommended)
+
+- Create and activate a brand new enviroment
+```
+conda create -n human_model_env
+conda activate human_model_env
+```
+- Install `idyntree` following [these instructions](https://github.com/robotology/idyntree?tab=readme-ov-file#conda-recommended) 
+- Install `urdf-modifiers`
+```
+git clone https://github.com/icub-tech-iit/urdf-modifiers.git
+cd urdf-modifiers
+pip install .
+```
 
 ## Usage
-- Clone this repository
-  ```
-  git clone https://github.com/ami-iit/human-model-generator.git
-  cd human-model-generator
-  ```
-- Open the file `src/generateModels.py` with a text editor.
-- Manually modify the parameters (see file [INFO.md](src/INFO.md)).
-- Generate the model by running on the terminal `python src/generateModels.py`.
-- A template called `FileName.urdf` will be saved in the folder `models/humanModel`.
+```
+git clone https://github.com/ami-iit/human-model-generator.git
+cd human-model-generator/code
+```
+- Open the file `config.py` with a text editor
+- Manually modify the parameters according to the human subject anthropometric measurements (see [this file](https://github.com/ami-iit/human-model-generator/blob/ReorganizeCode/code/README.md))
+- Generate the model by running `python main.py` 
+- A URDF model called `FileName.urdf` will be saved in the folder `models/humanModel`
```

Comment:
Actually this part of the text  is no longer true: 

> A URDF model called `FileName.urdf`

It's more generically like: The URDF model will be saved in the folder `models/humanModel`

Reply:
Done